packages/office-ui-fabric-react/src/components/Breadcrumb/Breadcrumb.tsx

```diff
@@ -128,6 +128,7 @@ export class Breadcrumb extends BaseComponent<IBreadcrumbProps, IBreadcrumbState
         <Link
           className={ css('ms-Breadcrumb-itemLink', styles.itemLink) }
           href={ item.href }
+          aria-current={item.isCurrentItem ? 'page' : null}
```

Comment:
Note, aria-current is currently not valid aria markup (it's introduced as part of aria 1.1 which is not a finalized standard), it's fine in this case because worst case scenario the browser and/or screen reader will ignore that property and having it missing isn't a big deal